test_conformance/math_brute_force/main.cpp

```diff
@@ -97,28 +97,24 @@ cl_mem gInBuffer2 = NULL;
 cl_mem gInBuffer3 = NULL;
 cl_mem gOutBuffer[VECTOR_SIZE_COUNT] = { NULL, NULL, NULL, NULL, NULL, NULL };
 cl_mem gOutBuffer2[VECTOR_SIZE_COUNT] = { NULL, NULL, NULL, NULL, NULL, NULL };
-uint32_t gComputeDevices = 0;
-uint32_t gSimdSize = 1;
-uint32_t gDeviceFrequency = 0;
 static MTdata gMTdata;
 cl_device_fp_config gFloatCapabilities = 0;
-cl_device_fp_config gDoubleCapabilities = 0;
```

Comment:
Think we could actually make this function scope where `CL_DEVICE_DOUBLE_FP_CONFIG` is queried